java/src/jmri/jmrit/signalling/SignallingSourcePanel.java

```diff
@@ -180,7 +169,10 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
             boolean newValue = (Boolean) e.getNewValue();
             discoverPairs.setEnabled(newValue);
         }
-        //log.debug("SSP 179 Event: {}", e.getPropertyName());
+        log.debug("SSP 173 Event: {}; Source: {}", e.getPropertyName(), e.toString()); // doesn't get notified, newDestination
+        if (e.getPropertyName().equals("Frame.active")) { // a blunt way of redrawing the Pairs table
```

Comment:
Listening for an SML Pair Edit to finish and redraw the displayed list is now working but it proved quite hard to receive an exact message from the Edit pane as there is no direct relationship. Frame.active will occur more often so this is not resource friendly, but for now it is the only message showing up in the debug console.
An alternative would be a Redraw button on the pane and wait for the user worrying his new pair is not ther, but that method seemed too inelegant to consider.